src/libsync/account.h

```diff
@@ -68,6 +68,10 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject {
 
     AccountPtr sharedFromThis();
 
+    /// The user that can be used in dav url
```

Comment:
I don't know.  I was just told it could be different and that i had to do an API call even when using the HTTPCrendentials.

@DeepDiver1975   Can you clarify?